sdk/highlight-go/otel.go

```diff
@@ -38,23 +39,6 @@ const MetricEvent = "metric"
 const MetricEventName = "metric.name"
 const MetricEventValue = "metric.value"
 
-var InternalAttributePrefixes = []string{
-	DeprecatedProjectIDAttribute,
-	DeprecatedSessionIDAttribute,
-	DeprecatedRequestIDAttribute,
-	DeprecatedSourceAttribute,
-	ProjectIDAttribute,
-	SessionIDAttribute,
-	RequestIDAttribute,
-	SourceAttribute,
-	LogMessageAttribute,
-	LogSeverityAttribute,
-	// exception should be parsed as structured and not included as part of log attributes
-	"exception.message",
-	"exception.stacktrace",
-	"fluent.",
```

Comment:
@Vadman97 - The code previously removed all `fluent.*` data. Not sure why we would want to do that (it could be useful?). I can re-add this logic back in if we need it.

Reply:
`fluent.tag` was the one we would extract and keep. don't think there were any other attributes we need to drop (https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/b847fe7f95da3c925d75dafac5559a13bfa12e78/receiver/fluentforwardreceiver/conversion.go#L19C26-L19C36), so it's safe to remove the exclusion.

Reply:
Gotcha. We are omitting that already so should be good 👍 
https://github.com/highlight/highlight/blob/7fb1e9b263e8e6d581ea5885934f889f144aa82f/backend/otel/extract.go#L189-L190